bikeshed/update/main.py

```diff
@@ -38,6 +38,7 @@ def update(
             say("Falling back to a manual update...")
             force = True
     if force:
+        # fmt: off
```

Comment:
I disabled this chunk because I think you want to keep this style of lining up the operators